app/assets/javascripts/admin/user/user_list_view.js

```diff
@@ -166,6 +168,29 @@ class UserListView extends React.PureComponent<Props, State> {
     }
   };
 
+  renderPlaceholder() {
+    const noUsersMessage = (
+      <React.Fragment>
+        {"You can "}
+        <a onClick={() => this.setState({ isInvitePopoverVisible: true })}>invite other users</a>
+        {" to join your organization. After users joined, you'll have to activate them manually."}
+      </React.Fragment>
+    );
+    const inactiveUsersMessage =
+      'There are users that have not been activated yet. Make sure the "Show Active Users Only" tag is removed to show those users and activate them.';
+    const inactiveUsersPresent = this.state.users.filter(user => !user.isActive).length > 0;
```

Comment:
Do we have the creation date of the users here? Then, we can only show inactive users which were created in the last two weeks or so.

Reply:
Otherwise, this will always show up as soon as there is one deactivated user.

Reply:
absolutely true, I can add the creation date to the json for you here